ChangeLog.md

```diff
@@ -1,6 +1,32 @@
 
 # Change Log
 
+## Version 10.18.0
+
+### New features
+
+1. Added support for `Content-MD5` in `list` command. User can now list the MD5 hash of the blobs in the target container.
+2. Added support to resume incomplete blobs. User can now resume the upload of a blob which was interrupted in the middle.
+3. Added support for download of POSIX properties.
+4. Added support for persisting symlink data.
+
+### Bug fixes
+
+1. Fixed [Issue 2120](https://github.com/Azure/azure-storage-azcopy/pull/2120)
+2. Fixed [Issue 2062](https://github.com/Azure/azure-storage-azcopy/pull/2062)
+3. Fixed [Issue 2046](https://github.com/Azure/azure-storage-azcopy/pull/2048)
+4. Fixed [Issue 1762](https://github.com/Azure/azure-storage-azcopy/pull/2125)
+
```

Comment:
Could you please add a line for this PR: https://github.com/Azure/azure-storage-azcopy/pull/2125?

Reply:
Actually, that's already present on the last line-- I was just referencing the actual issue mentioned.

Knew I'd got that one.

Reply:
Oh! Thank you, didn't notice that.